web/stories/ui/navbar.stories.svelte

```diff
@@ -51,6 +51,17 @@
   </Scene>
 </Story>
 
+<Story
```

Comment:
We may need also a story for small tablets. How menu will look like on 800px?

Reply:
Should I make a story for tablets even if the design for tablets is the same as for mobile? Or it would be enough that inside the mobile story you can change a viewport?

Reply:
> Should I make a story for tablets even if the design for tablets is the same as for mobile?

Yes. Because we need to check how this design will scale to tablet.

For instance, we can add spacers around Fast/Slow or stretch those buttons (I am not sure that it will look good).

And we need to be sure that we will not have regression in design (broken spacers).

Also, later we will have a little changes for tablet UI.

Reply:
For now I've added a story for a tablet but I haven't experimented with design yet.

Reply:
This is a good thing about stories (separated stories for each case, and not changing viewport in Storybook).

It forces thinking about edge cases.